### ISSUE TITLE
Does this still work when used in ASP.NET Core 2 in Linux or MacOS?

### ISSUE THREAD

**Issue by whizkidwwe1217** [bcgit/bc-csharp#139]

**Comment by qmfrederik** [comment]
Yes it does, although you may want to use Portable.BouncyCastle for that.

**Comment by MisterJimson** [comment]
Perhaps the main package should be updated to install the correct implementation for the target project instead of using multiple packages. This should be doable with multi targeting.

**Comment by jstedfast** [comment]
That's what Portable.BouncyCastle is: https://github.com/bcgit/bc-csharp/pull/68
